src/uk/gov/defra/ffc/Utils.groovy

```diff
@@ -60,12 +60,7 @@ class Utils implements Serializable {
     if (!errMessage) {
       def errCauses = e.getCauses()
       if (errCauses) {
-        def errMessages = []
-        errCauses.each { errCause ->
-          if (errCause instanceof io.snyk.jenkins.workflow.FoundIssuesCause) {
-            errMessages.add(errCause.getShortDescription())
-          }
-        }
+        def errMessages = []        
```

Comment:
I don't think the majority of this code is actually doing anything now Snyk has been removed.

```
static def getErrorMessage(e) {
    def errMessage = e.message  // try to get the message from the error
    if (!errMessage) {  // if there is no error message attached
      def errCauses = e.getCauses() // get the causes instead
      if (errCauses) { // if there are any causes
        def errMessages = []     // create an empty array
        errMessage = errMessages.join(', ') // error message now equals the empty array - this will never have a value
      }
      errMessage = errMessage ?: 'No error message available.'  // replace the error message with the empty array if it's not empty (it always will be) or the phrase no message available
    }
    return errMessage // return the error message
  }
```

So the error message will either be the message attached to the error or no error available.

You could simplify this method to just:

```
static def getErrorMessage(e) {
    return e.message ? e.message : 'No error message available'
 }
```

Reply:
Agreed.